src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/requireemptylinebeforeblocktaggroup/Example1.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RequireEmptyLineBeforeBlockTagGroup"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.requireemptylinebeforeblocktaggroup;
+
+// xdoc section -- start
+class Example1 {
+  /**
+   * ValidMethod's javadoc.
+   *
+   * @return something
+   */
+  boolean ValidMethod() {
```

Comment:
methodWithInvalidJavadoc

Please apply same naming pattern to other method

Reply:
Done